src/OpenApi/src/Extensions/OpenApiSchemaExtensions.cs

```diff
@@ -18,4 +18,21 @@ public static IOpenApiSchema CreateOneOfNullableWrapper(this IOpenApiSchema orig
             ]
         };
     }
+
+    public static bool IsComponentizedSchema(this OpenApiSchema schema)
+        => schema.IsComponentizedSchema(out _);
+
+    public static bool IsComponentizedSchema(this OpenApiSchema schema, out string schemaId)
+    {
+        if(schema.Metadata is not null
+            && schema.Metadata.TryGetValue(OpenApiConstants.SchemaId, out var schemaIdAsObject)
+            && schemaIdAsObject is string schemaIdString
+            && !string.IsNullOrEmpty(schemaIdString))
+        {
+            schemaId = schemaIdString;
+            return true;
+        }
+        schemaId = string.Empty;
```

Comment:
Just curious, why does this return empty when the other extension of the same name returns null?